syntax = "proto3";
import "Ip.proto";
package rpc;

message RegistrationRequest
{
    repeated string requestedService=1;
    repeated string providedService=2;
    Ip nginxIp=3;
}

message RegistrationResponse
{
    repeated Ip ip=1;
    repeated string service=2;
}